tox.ini

```diff
@@ -209,15 +139,15 @@ commands =
 description = generate coverage report
 
 depends =
-    coverage-py{36,37,38,39,310,py3}-tw{1,2}{0,1,2,3,4,5,6,7,8,9}{0,1,2,3,4,5,6,7,8,9}
-    coverage-py{36,37,38,39,310,py3}-tw{current,trunk}
+    coverage-py{37,38,39,310,311,py3}-tw{1,2}{0,1,2,3,4,5,6,7,8,9}{0,1,2,3,4,5,6,7,8,9}
+    coverage-py{37,38,39,310,311,py3}-tw{current,trunk}
 
 basepython = {[default]basepython}
 
 skip_install = True
 
 deps =
-    coverage
+    -r requirements/coverage.txt
```

Comment:
Very minor comment.
I think that this if fine, and we can merge it as it is.

My preference, is to define an `extras = coverage` or just a simple `extras = dev` and in setup.py define all the development/testing  related packages as an `extra_require`.
In this way, all the deps are available inside the same file... and it should be easier to review and discover the deps.

Reply:
That's inconsistent with the other tox environments.

I don't think need we need to expose tox test environments in setup.py as extras, so I would hesitant to move that change.

Reply:
Those dependencies are supposed to be used for local test run and CI test runs.

When doing local dev, I also want to run the coverage reports locally.